<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>463</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Parallel Coordinates View Configuration</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="dataConfTab" >
      <attribute name="title" >
       <string>Data Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>24</width>
           <height>55</height>
          </size>
         </property>
         <property name="title" >
          <string>Pick data location from graph : </string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3" >
          <property name="margin" >
           <number>1</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <property name="spacing" >
             <number>120</number>
            </property>
            <item>
             <widget class="QRadioButton" name="nodesButton" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>nodes</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="edgesButton" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>edges</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Select properties to plot : </string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4" >
          <property name="margin" >
           <number>1</number>
          </property>
          <item>
           <widget class="tlp::GraphPropertiesSelectionWidget" native="1" name="graphPropertiesSelectionWidget" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>groupBox_2</zorder>
      <zorder>groupBox</zorder>
     </widget>
     <widget class="QWidget" name="drawConfTab" >
      <attribute name="title" >
       <string>Draw Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="font" >
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="title" >
          <string>General draw parameters : </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <property name="margin" >
           <number>1</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" >
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font" >
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text" >
               <string>Background Color :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bgColorButton" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="palette" >
               <palette>
                <active>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" >
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font" >
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text" >
               <string>Axis height :  </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="axisHeight" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum" >
               <number>100</number>
              </property>
              <property name="maximum" >
               <number>10000</number>
              </property>
              <property name="value" >
               <number>400</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" >
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font" >
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text" >
               <string>Space between axis : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spaceBetweenAxis" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum" >
               <number>10</number>
              </property>
              <property name="maximum" >
               <number>10000</number>
              </property>
              <property name="value" >
               <number>200</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4" >
            <property name="title" >
             <string>Lines colors alpha value :</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3" >
             <property name="margin" >
              <number>1</number>
             </property>
             <item>
              <widget class="QRadioButton" name="viewColorAlphaRb" >
               <property name="font" >
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text" >
                <string>viewColor value</string>
               </property>
               <property name="checked" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5" >
               <item>
                <widget class="QRadioButton" name="userAlphaRb" >
                 <property name="font" >
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text" >
                  <string>user value                                                                                 </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="viewColorAlphaValue" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum" >
                  <number>255</number>
                 </property>
                 <property name="value" >
                  <number>200</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="font" >
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text" >
               <string>Non highlighted elements colors alpha value :                                 </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="nonHighlightedEltsAlphaValue" >
              <property name="maximum" >
               <number>255</number>
              </property>
              <property name="value" >
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gBoxAxisPoints" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="title" >
          <string>Draw points on axis :                                                                                      </string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <property name="verticalSpacing" >
           <number>0</number>
          </property>
          <property name="margin" >
           <number>1</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_7" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font" >
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Minimum axis point size : </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QSpinBox" name="minAxisPointSize" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>66</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="value" >
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_8" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font" >
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Maximum axis point size : </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="maxAxisPointSize" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="value" >
             <number>15</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gBoxLineTexture" >
         <property name="font" >
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="title" >
          <string>Apply texture on lines :                                                                                  </string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <property name="margin" >
           <number>1</number>
          </property>
          <item>
           <widget class="QRadioButton" name="defaultTexture" >
            <property name="font" >
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Use tulip default texture</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="userTexture" >
            <property name="font" >
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Use texture from file :</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QLineEdit" name="userTextureFile" />
            </item>
            <item>
             <widget class="QPushButton" name="browseButton" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset>
                <normalon>:/i_open.png</normalon>
               </iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tlp::GraphPropertiesSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global" >tulip/GraphPropertiesSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ParallelResource.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>280</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel" >
     <x>234</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>396</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
